src/CommunityToolkit.Maui.Core/Views/DrawingView/Service/DrawingViewService.android.cs

```diff
@@ -0,0 +1,223 @@
+﻿using Android.Graphics;
+using Microsoft.Maui.Platform;
+using Color = Microsoft.Maui.Graphics.Color;
+using Math = System.Math;
+using Paint = Android.Graphics.Paint;
+using Point = Microsoft.Maui.Graphics.Point;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// Drawing view service
+/// </summary>
+public static partial class DrawingViewService
+{
+	/// <summary>
+	/// Get image stream from lines
+	/// </summary>
+	/// <param name="lines">Drawing lines</param>
+	/// <param name="imageSize">Image size</param>
+	/// <param name="backgroundColor">Image background color</param>
+	/// <returns>Image stream</returns>
+	public static Stream GetImageStream(in IList<ILine> lines,
+										in Size imageSize,
+										in Color backgroundColor)
+	{
+
+		var image = GetBitmapForLines(lines, backgroundColor);
+		if (image is null)
+		{
+			return Stream.Null;
+		}
+
+		using var resizedImage = GetMaximumBitmap(image, (float)imageSize.Width, (float)imageSize.Height);
+		var stream = new MemoryStream();
+		var compressResult = resizedImage.Compress(Bitmap.CompressFormat.Jpeg, 100, stream);
+
+		resizedImage.Recycle();
+
+		if (!compressResult)
+		{
+			return Stream.Null;
+		}
+
+		stream.Position = 0;
+		return stream;
+	}
+
+	/// <summary>
+	/// Get image stream from points
+	/// </summary>
+	/// <param name="points">Drawing points</param>
+	/// <param name="imageSize">Image size</param>
+	/// <param name="lineWidth">Line Width</param>
+	/// <param name="strokeColor">Line color</param>
+	/// <param name="backgroundColor">Image background color</param>
+	/// <returns>Image stream</returns>
+	public static Stream GetImageStream(in IList<Point> points,
+										in Size imageSize,
+										in float lineWidth,
+										in Color strokeColor,
+										in Color backgroundColor)
+	{
+		if (points.Count < 2)
+		{
+			return Stream.Null;
+		}
+
+		var image = GetBitmapForPoints(points, lineWidth, strokeColor, backgroundColor);
+		if (image is null)
+		{
+			return Stream.Null;
+		}
+
+		using var resizedImage = GetMaximumBitmap(image, (float)imageSize.Width, (float)imageSize.Height);
+		var stream = new MemoryStream();
+		var compressResult = resizedImage.Compress(Bitmap.CompressFormat.Jpeg, 100, stream);
+
+		resizedImage.Recycle();
+
+		if (!compressResult)
+		{
+			return Stream.Null;
+		}
+
+		stream.Position = 0;
+		return stream;
+	}
+
+	static Bitmap? GetBitmapForPoints(ICollection<Point> points,
+		float lineWidth,
+		Color strokeColor,
+		Color backgroundColor)
+	{
+		if (!points.Any())
```

Comment:
if I'm not wrong `ICollection` implements `Count` prop.
```suggestion
		if (points.Count is 0)
```